person.rb

```diff
@@ -1,16 +1,21 @@
 require './decorator'
 
 class Person < Nameable
-  attr_accessor :id, :name, :age
+  attr_accessor :id, :name, :age, :rentals
 
   def initialize(age, name = 'Unknown')
     @age = age
     @name = name
     @id = id
     @parent_permission = true
+    @rentals = []
     super()
   end
 
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] [OPTIONAL] Please kindly ensure that the person parameter is changed to book or the method should be invoked with book instead of person.